### PR TITLE
Update apm publish npm scripts

### PR DESCRIPTION
**Pull request by galactusss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ Running your app using IPFS will mimic the production environment that will be u
 - **build:app**: Builds the front-end
 - **build:script**: Builds the background script
 - **test**: Runs tests for the contracts
-- **publish**: Builds the apps and the contracts and publishes them to IPFS and APM
+- **publish:content**: Builds the apps publishes to IPFS
+- **publish:minor**: Builds the apps and the contracts and publishes them to IPFS and APM
+- **publish:major**: Builds the apps and the contracts and publishes them to IPFS and APM. Deployed contract will default to the contract name specified in the arapp.json file
```
Comment (sohkai) [inline README.md]: I'd reword these comments:

- `publish:content`: Release a minor version to aragonPM with just the new content
- `publish:minor`: Release a minor version to aragonPM
  - ⚠️ **warning**: since we only allow contract upgrades on major versions, isn't this the same as `publish:content`?
- `publish:major`: Release a major version to aragonPM with a potentially new contract address for on-chain upgrades.
  - ⚠️**warning**: if the contract doesn't change, does aragonCLI automatically reuse the same contract address or will it always redeploy the contract?

File: arapp.json [hunk 2]
```diff
@@ -13,5 +11,11 @@
       "params": []
     }
   ],
-  "path": "contracts/CounterApp.sol"
+  "path": "contracts/CounterApp.sol",
```
Comment (sohkai) [inline arapp.json]: Would put `path` last and `environments` first, to sort the keys by alphabetical order.

File: arapp.json [hunk 3]
```diff
@@ -13,5 +11,11 @@
...
+  "environments": {
+    "default": {
+      "network": "development",
+      "appName": "app.aragonpm.eth"
```
Comment (sohkai) [inline arapp.json]: We should make sure aragonCLI probably changes the name of this on `aragon init` (not sure if this logic was added in as part of the change to make `environments` happen.

Comment (galactusss) [reply arapp.json]: Yes, this was added as part of the changes

File: arapp.json (hunk 2 above)
Comment (galactusss) [reply arapp.json]: Ok, do you think we should change this also on aragonCLI. Right now `aragon init` use this order

File: README.md (hunk 1 above)
Comment (galactusss) [reply README.md]: You are right `publish:minor` and `publish:content` are doing the same. About major, right now, aragonCLI is creating new address every time even if contract its the same
